html/semantics/links/following-hyperlinks/activation-behavior.window.js

```diff
@@ -0,0 +1,43 @@
+["a",
+ "area"].forEach(type => {
```

Comment:
Missing "link"

Also isn't this file partially redudant with active-document.window.js, introduced in https://github.com/w3c/web-platform-tests/pull/5758 ?

Reply:
The `link` element doesn't have `target`.

Reply:
I don't think it's redundant. That's a link in a document that's navigated away. Here it's a link that's simply not connected (or from a synthetic document).